rtrlib/lib/vector.h

```diff
@@ -0,0 +1,58 @@
+/*
+* This file is part of RTRlib.
+*
+* This file is subject to the terms and conditions of the MIT license.
+* See the file LICENSE in the top level directory for more details.
+*
+* Website: http://rtrlib.realmv6.org/
+*/
+
+#ifndef LRTR_VECTOR_UTILS_H
+#define LRTR_VECTOR_UTILS_H
+
+#include <stdint.h>
```

Comment:
are these includes really required? i think not

Reply:
yoy need `stdint` for uint32_t

Reply:
what about string.h?

Reply:
required for memcpy

Reply:
sorry, thanks